src/load_inputs/load_minimum_capacity_requirement.jl

```diff
@@ -28,5 +28,11 @@ function load_minimum_capacity_requirement!(path::AbstractString, inputs::Dict,
     if setup["ParameterScale"] == 1
         inputs["MinCapReq"] /= ModelScalingFactor # Convert to GW
     end
+    if "PriceCap" in names(df)
+        inputs["MinCapPriceCap"] = df[!,:PriceCap]
+        if setup["ParameterScale"] == 1
```

Comment:
This function now has two `if setup["ParameterScale"]` blocks. We could replace both with
`scale_factor = setup["ParameterScale"] == 1 ? ModelScalingFactor : 1`.

Reply:
Yeah, that would be a useful change.